kubernetes/asanaban-service-deployment.yml

```diff
@@ -0,0 +1,64 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: asanaban-service
+  namespace: apps
+spec:
+  selector:
+    app: asanaban
+  ports:
+    - name: http
+      protocol: TCP
+      port: 3000
+      targetPort: 3000 # Note this needs to match the secrets
+  type: ClusterIP
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: asanaban
+  namespace: apps
+  labels:
+    name: asanaban
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: asanaban
+  minReadySeconds: 5
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 0
+      maxSurge: 1
+  template:
+    metadata:
+      labels:
+        app: asanaban
+    spec:
+      containers:
+        - name: asanaban
+          image: gcr.io/searchspring-devops/asanaban:latest
+          imagePullPolicy: IfNotPresent
+          ports:
+            - containerPort: 3000
+          startupProbe: # Startup probe makes newly deployed pod wait for cache refresh to finish before marking it as ready
```

Comment:
This comment is likely copy/pasta mistake. I don't think you need the probe at all